managed/services/agents/jobs.go

```diff
@@ -227,7 +227,42 @@ func (s *JobsService) handleJobResult(_ context.Context, l *logrus.Entry, result
 				return errors.Errorf("result type %s doesn't match job type %s", models.MongoDBRestoreBackupJob, job.Type)
 			}
 
-			_, err := models.ChangeRestoreHistoryItem(
+			logChunkID := 0
+			var logParams models.CreateJobLogParams
+			if job.Data.MongoDBRestoreBackup.DataModel == models.PhysicalDataModel {
+				logParams = models.CreateJobLogParams{
+					JobID:     job.ID,
+					ChunkID:   logChunkID,
+					LastChunk: false,
+					Data:      "restore successfully completed, PMM will restart mongod and pbm-agent.",
+				}
+			} else {
+				logParams = models.CreateJobLogParams{
+					JobID:     job.ID,
+					ChunkID:   logChunkID,
+					LastChunk: true,
+					Data:      "restore successfully completed.",
+				}
+			}
+			if err := createJobLog(t.Querier, logParams.JobID, logParams.Data, logParams.ChunkID, logParams.LastChunk); err != nil {
+				s.l.WithError(err).Errorf("failed to create log for job %s [chunk: %d]", job.ID, logChunkID)
+			}
```

Comment:
In that case we can read `logChunkID`  from the DB (i.e last chunk that was saved) instead of starting from 0

Reply:
Yep, that's wrong. Chunk ID counter defined on the agent side, and can be incremented only there. Moreover logs chunks can arrive on PMM server after job result. In your implementation you will have chunk ID collision because you adding something there on server side.

Reply:
update on this: we eventually agreed to keep it this way since we don't stream restore logs yet, and re-work logs later.